packages/venia-concept/src/shared/images.js

```diff
@@ -1,5 +1,7 @@
 // corresponds to a 300x372 transparent png
 // keep in sync with constants above
 // TODO: generate this programmatically?
-export const imagePlaceholderUri =
+export const transparentPlaceholder =
     'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABkAAAAfCAQAAAC4ua71AAAAGklEQVR42mNkIBkwjmoZ1TKqZVTLqJYRpgUAaP0AIAQAObYAAAAASUVORK5CYII=';
+
+export const grayPlaceholder = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAAFCAQAAADIpIVQAAAADklEQVR42mNkgAJGIhgAALQABsHyMOcAAAAASUVORK5CYII=';
```

Comment:
None of the original placeholders were gray; they were all transparent. Have you changed one of them to gray or is this misnamed?

Reply:
Looks like I made a bad assumption based on some CSS, and the fact that those two base64 strings are slightly different. But examining them...they're both transparent and they only differ by size. 
![image](https://user-images.githubusercontent.com/1643758/42003241-14b5a416-7a30-11e8-9a20-29c010a15bba.png)
![image](https://user-images.githubusercontent.com/1643758/42003254-20e621c0-7a30-11e8-8c4e-d19daa8a3f20.png)

Whoops! I'll just use the one `transparentPlaceholder`.